css/CSS2/backgrounds/background-root-017.xht

```diff
@@ -9,7 +9,6 @@
   <link rel="help" href="http://www.w3.org/TR/CSS21/colors.html#background"/>
   <link rel="match" href="background-root-017-ref.xht" />
 
-  <meta name="flags" content="interact"/>
```

Comment:
Please use `<meta name="flags" content="" />` rather than deleting the line. See the comment on focus-within-001 for rationale.